examples/pytorch/hgp_sl/README.md

```diff
@@ -0,0 +1,115 @@
+# DGL Implementation of the HGP-SL Paper
+
+This DGL example implements the GNN model proposed in the paper [Hierarchical Graph Pooling with Structure Learning](https://arxiv.org/pdf/1911.05954.pdf). 
+The author's codes of implementation is in [here](https://github.com/cszhangzhen/HGP-SL)
+
+
+Example implementor
+----------------------
+This example was implemented by [Tianqi Zhang](https://github.com/lygztq) during his Applied Scientist Intern work at the AWS Shanghai AI Lab.
+
+
+The graph dataset used in this example 
+---------------------------------------
+The DGL's built-in LegacyTUDataset. This is a serial of graph kernel datasets for graph classification. We use 'DD', 'PROTEINS', 'NCI1', 'NCI109', 'Mutagenicity' and 'ENZYMES' in this HGP-SL implementation. All these datasets are randomly splited to train, validation and test set with ratio 0.8, 0.1 and 0.1.
```

Comment:
Better add a link to the doc of LegacyTUDataset

Reply:
solved